spectralcluster/utils.py

```diff
@@ -39,7 +40,7 @@ def compute_sorted_eigenvectors(A):
            eigenvalue
     """
     # Eigen decomposition.
-    eigenvalues, eigenvectors = np.linalg.eig(A)
+    eigenvalues, eigenvectors = scipy.linalg.eigh(A, turbo=True)
```

Comment:
Also, just curious, is `scipy.linalg.eig` faster than `numpy.linalg.eig`? Do you have any reference for this?

Reply:
scipy.linalg.eig is not faster than numpy.linalg.eig. This request is meaningless.